src/main/java/com/feywild/feywild/sound/ModSoundEvents.java

```diff
@@ -35,4 +35,12 @@
     public static final SoundEvent summoningSummerPixieShort = new SoundEvent(new ResourceLocation(FeywildMod.getInstance().modid, "summoning_summer_pixie_short"));
     public static final SoundEvent summoningAutumnPixieShort = new SoundEvent(new ResourceLocation(FeywildMod.getInstance().modid, "summoning_autumn_pixie_short"));
     public static final SoundEvent summoningWinterPixieShort = new SoundEvent(new ResourceLocation(FeywildMod.getInstance().modid, "summoning_winter_pixie_short"));
+    public static final SoundEvent shroomlingAmbience01 = new SoundEvent(new ResourceLocation(FeywildMod.getInstance().modid, "shroomling_ambience_01"));
+    public static final SoundEvent shroomlingAmbience02 = new SoundEvent(new ResourceLocation(FeywildMod.getInstance().modid, "shroomling_ambience_02"));
+    public static final SoundEvent shroomlingDeath = new SoundEvent(new ResourceLocation(FeywildMod.getInstance().modid, "shroomling_death"));
+    public static final SoundEvent shroomlingHurt = new SoundEvent(new ResourceLocation(FeywildMod.getInstance().modid, "shroomling_hurt"));
+    public static final SoundEvent shroomlingSneeze = new SoundEvent(new ResourceLocation(FeywildMod.getInstance().modid, "shroomling_sneeze"));
+    public static final SoundEvent shroomlingWave01 = new SoundEvent(new ResourceLocation(FeywildMod.getInstance().modid, "shroomling_wave_01"));
+    public static final SoundEvent shroomlingWave02 = new SoundEvent(new ResourceLocation(FeywildMod.getInstance().modid, "shroomling_wave_02"));
```

Comment:
Instead of creating a random number and then playing one of the two sound events, you can add both sounds to the same event in `sounds.json`

Reply:
Changed other SoundEvents too like Mandragora
Added a boolean returning null so the sounds aren't constantly playing